pydantic/deprecated/json.py

```diff
@@ -70,11 +73,13 @@ def decimal_encoder(dec_value: Decimal) -> Union[int, float]:
 }
 
 
+@deprecated('pydantic_encoder is deprecated, use BaseModel.model_dump instead.')
```

Comment:
```suggestion
@deprecated('pydantic_encoder is deprecated, use `pydantic_core.to_jsonable_python` instead.')
```
I think this is the better substitute. (And I think addresses @adriangb's point as well)